Core/Core/Transports/ITransport.cs

```diff
@@ -90,9 +96,9 @@ public interface ITransport
   /// <param name="targetTransport">The transport you want to copy the object to.</param>
   /// <param name="onTotalChildrenCountKnown">(Optional) an <see cref="Action{T}"/> that will be invoked once, when the number of object children to be copied over is known.</param>
   /// <returns>The string representation of the root object.</returns>
-  /// <exception cref="InvalidOperationException">The transport was in an invalid state</exception>
   /// <exception cref="ArgumentException">The provided arguments are not valid</exception>
-  /// <exception cref="OperationCanceledException"></exception>
+  /// <exception cref="InvalidOperationException">The transport was in an invalid state</exception>
+  /// <exception cref="OperationCanceledException"><see cref="CancellationToken"/> requested cancel</exception>
```

Comment:
Again, beyond scope but the commented out GetBlock() call below? Worth binning it while we are here?

Reply:
Depends whether you consider this a comment to stop developers adding a `GetBlob` function back later down the road.
I'll leave this one up to Gergo